### PR TITLE
Edit user profile

### PR DESCRIPTION
**Pull request by tanishk15sharma**
- [x] Edit `useState`
- [x] upload profile image
- [x] edit REST API call
- [x] Delete button
- [x] Delete user REST API call

### CODE REVIEW

**Review by Hardiegogo — APPROVED** [review]
Good to merge.✔

File: src/components/EditModal.js [hunk 1]
```diff
@@ -11,8 +40,20 @@ const EditModal = ({ setToggleEditModal }) => {
...
             T{/* <img src={profileBg} alt="profile-bg" /> */}
+            <label htmlFor="imageFile" className="cursor-pointer">
+              <span className="material-icons-outlined absolute z-20 opacity-5 hover:opacity-80 text-6xl">
+                photo_camera
+              </span>
+            </label>
+            <input
+              type="file"
+              id="imageFile"
+              accept=".png,.jpeg,.jpg"
```
Comment (znjs) [inline src/components/EditModal.js:53]: you can add these formats also `image/apng, image/avif, image/gif, image/jpeg, image/png, image/svg+xml, image/jpg,image/webp`

**Review by znjs — APPROVED** [review]
- Good PR
- Clean code

File: src/components/EditModal.js (hunk 1 above)
Comment (tanishk15sharma) [reply src/components/EditModal.js:53]: sure

**Pull request merged**